README.md

```diff
@@ -1,9 +1,13 @@
-Welcome to Legato!
+![Legato logo](http://legato.io/resources/img/legato_logo.png)
+ 
 ==================
 
 ![Build Status](https://travis-ci.org/legatoproject/legato-af.svg)
 
-#### Dependencies Alan
+Legato Open Source Project started out as an initiative by Sierra Wireless Inc. to provide an open, secure and easy to use Application Framework to lower the barrier for developers, who are not familiar with embedded programming, to participate in the exponential growth of "Internet of Things".
+Visit [legato.io](legato.io) to learn more or visit the [Legato forum](forum.legato.io).
```

Comment:
The link URLs need to be complete or else they will link to something relative to this page.  So change to \[legto.io\](http://legato.io) for example.